resources/js/Pages/Profiles/Index.tsx

```diff
@@ -1,34 +1,89 @@
 import React from "react";
 import { InertiaLink } from "@inertiajs/inertia-react";
 import FormLayout from "@/Atoms/FormLayout";
+import ProfileMenteeSection from "@/Organisms/ProfileMenteeSection";
+import ProfileMentorSection from "@/Organisms/ProfileMentorSection";
 
-interface Props {
+export interface IProps {
     heading?: string;
+    mentor_profile?: {
```

Comment:
The normal naming convention for javascript objects is  camelCase so it might be worth renaming these to `mentorProfile` and `menteeProfile` for consistency. Up to you 🙂